karaf/init/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/osgi/ToscaTypePlanTransformerClassloaderWrapper.java

```diff
@@ -17,7 +20,7 @@
  * classloader for this bundle, rather than whatever may happen to be in place at the
  * point of call.
  */
-public class ToscaTypePlanTransformerClassloaderWrapper implements BrooklynTypePlanTransformer {
+public class ToscaTypePlanTransformerClassloaderWrapper extends ToscaTypePlanTransformer implements BrooklynTypePlanTransformer {
```

Comment:
does this inherit any methods from `TTPT`?  if it's pure delegate a marker interface might be cleaner, or at least a note that the inheritance is purely for injectability purposes.